terraform/main.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  backend "remote" {
+    organization = "tinkerbell"
+
+    workspaces {
+      name = "github_management"
+    }
+  }
+}
+
+provider "github" {
+  token        = var.github_token
```

Comment:
this variable is no longer defined, so I expect this to fail

Reply:
https://www.terraform.io/docs/providers/github/index.html#token - it is optional, we can remove it.